README.md

```diff
@@ -48,131 +44,6 @@ bundle install
    - To access the sidekiq dashboard, first sign into the application, then go to http://localhost:3000/sidekiq
 1. To stop: `bundle exec rake servers:stop` or `lando stop`
 
-## Alma
-
-### Configure Alma keys for Development
-
-1. `lpass login emailhere`
-1. `bundle exec rake alma:setup_keys`
-
-This will add a .env with credentials to Rails.root
```

Comment:
It would be good to keep these steps in the README, as part of setting up a dev environment.

Reply:
Copied to README as I think it makes sense to have in both locations.